Akka.Persistence.Pulsar/Journal/PulsarJournal.cs

```diff
@@ -53,8 +53,11 @@ public PulsarJournal(PulsarSettings settings)
             long toSequenceNr, long max,
             Action<IPersistentRepresentation> recoveryCallback)
         {
-            var startMessageId =
-                new MessageId(ledgerId, entryId, partition, batchIndex); //TODO: how to config them properly in Pulsar?
+            //according to pulsar doc, messageid is returned for each message produced. The Pulsar system is in charge of creating MessageId
```

Comment:
Are you sure that we're not able to infer message id from akka-generated sequence nr?

Reply:
In MongoDb, JournalEntry is created to hold this info!
Though Pulsar support SQL-like query using Presto, that could be one option!
What we are dealing with here is "WHERE TO START FROM".
The Sequence Nr of Pulsar is of type ulong while akka's is long.

Am still thinking though.....

Reply:
https://github.com/streamnative/pulsar/issues/433

Reply:
https://github.com/apache/pulsar/issues/3180

Reply:
I pushed a commit that tends to use SequenceNr to build MessageId!

Reply:
I pushed a commit that tends to use SequenceNr to build MessageId!